test/utils/throwingInstance.ts

```diff
@@ -0,0 +1,24 @@
+import { instance } from "ts-mockito";
+
+/**
+ * This will be useful for detecting when certain methods have not been stubbed.
+ * @param target will take the mocked object
+ */
+export default function throwingInstance(target: any) {
```

Comment:
This function returns `any`. In doing so we lose the type annotations when we pass an object through throwingInstance. This function shouldn't expect any - it should expect a generic type T - which you should use wherever possibly in this function, including the return type.